run.sh

```diff
@@ -48,9 +48,19 @@ for vin in $VIN_LIST; do
   if [ -f $KEYS_DIR/private.pem ] && [ $vin_count -eq 1 ]; then
     log_warning "Keys exist from a previous installation with single VIN which is deprecated"
     log_warning "This module migrates the key files to attribute them to $vin and remove old MQTT entities"
-    log_warning "$KEYS_DIR/private.pem $KEYS_DIR/${vin}_private.pem"
-    log_warning "$KEYS_DIR/public.pem $KEYS_DIR/${vin}_public.pem"
-    delete_legacies $vin
+
+    log_warning "rename $KEYS_DIR/private.pem -> $KEYS_DIR/${vin}_private.pem"
+    mv $KEYS_DIR/private.pem $KEYS_DIR/${vin}_private.pem
+    log_warning "rename $KEYS_DIR/public.pem -> $KEYS_DIR/${vin}_public.pem"
+    mv $KEYS_DIR/public.pem $KEYS_DIR/${vin}_public.pem
+
+    delete_legacies
+    delete_legacies_singles ""
```

Comment:
we need to pass `${vin}` there?

Reply:
No that's the idea... if there's no vin it will delete tesla_ble/ instead of tesla_ble${add_vin}  and add_vin = "_${vin}".